  INFO     Context: summary changed [conclusions: {'Rule: queueit (queue)': <Conclusion.SUCCESS: 'success'>}] [gh_author: mergify-sileht[bot]] [gh_branch: 20231018114631/test_freeze_eventlogs/main] [gh_owner: mergifyio-testing] [gh_private: False] [gh_pull: 33773] [gh_pull_base_sha: d9b44ca14ce953bf19608307ae8e0f9eaad0083e] [gh_pull_head_owner: mergifyio-testing] [gh_pull_head_ref: 20231018114631/test_freeze_eventlogs/integration/pr1] [gh_pull_head_sha: 8367f370bd775373f19a7251a28a0ec5ed9de202] [gh_pull_locked: False] [gh_pull_merge_commit_sha: 25947d83d124983780050f108defc2c78998fe8b] [gh_pull_mergeable: True] [gh_pull_mergeable_state: unstable] [gh_pull_url: https://github.com/mergifyio-testing/functional-testing-repo-sileht/pull/33773] [gh_repo: functional-testing-repo-sileht] [previous_conclusions: {'Rule: queueit (queue)': <Conclusion.PENDING: None>}] [worker_id: shared-0]
  INFO     Context: actions runner [configuration_changed: False] [gh_author: mergify-sileht[bot]] [gh_branch: 20231018114631/test_freeze_eventlogs/main] [gh_owner: mergifyio-testing] [gh_private: False] [gh_pull: 33773] [gh_pull_base_sha: d9b44ca14ce953bf19608307ae8e0f9eaad0083e] [gh_pull_head_owner: mergifyio-testing] [gh_pull_head_ref: 20231018114631/test_freeze_eventlogs/integration/pr1] [gh_pull_head_sha: 8367f370bd775373f19a7251a28a0ec5ed9de202] [gh_pull_locked: False] [gh_pull_merge_commit_sha: 4b0471b042006241d0f38f9d8d44e1657d39ed32] [gh_pull_mergeable: None] [gh_pull_mergeable_state: merged] [gh_pull_url: https://github.com/mergifyio-testing/functional-testing-repo-sileht/pull/33773] [gh_repo: functional-testing-repo-sileht] [sources: [{'event_type': 'refresh', 'data': {'action': 'internal', 'source': 'merged pull 33773 removed from queue', 'ref': None, 'pull_request_number': 33773, 'sender': {'login': '<internal>', 'id': 0, 'type': 'User'}, 'flag': None, 'attempts': None, 'delivery_id': None, 'received_at': '2023-10-20T09:16:28.237368+00:00'}, 'timestamp': '2023-10-20T09:16:28.237442+00:00', 'initial_score': 1016977933882374.0}]] [worker_id: shared-0]
  INFO     Context: ignored pull request rules [gh_author: mergify-sileht[bot]] [gh_branch: 20231018114631/test_freeze_eventlogs/main] [gh_owner: mergifyio-testing] [gh_private: False] [gh_pull: 33773] [gh_pull_base_sha: d9b44ca14ce953bf19608307ae8e0f9eaad0083e] [gh_pull_head_owner: mergifyio-testing] [gh_pull_head_ref: 20231018114631/test_freeze_eventlogs/integration/pr1] [gh_pull_head_sha: 8367f370bd775373f19a7251a28a0ec5ed9de202] [gh_pull_locked: False] [gh_pull_merge_commit_sha: 4b0471b042006241d0f38f9d8d44e1657d39ed32] [gh_pull_mergeable: None] [gh_pull_mergeable_state: merged] [gh_pull_url: https://github.com/mergifyio-testing/functional-testing-repo-sileht/pull/33773] [gh_repo: functional-testing-repo-sileht] [pull_request_rules: [{'name': 'delete backport/copy branch (Mergify rule)', 'rule_summary': '- [ ] `head~=^mergify/(bp|copy)/`\n- [X] `author=mergify-sileht[bot]`\n- [X] `closed` [:pushpin: delete_head_branch requirement]', 'actions': ['delete_head_branch']}]] [worker_id: shared-0]
  INFO     Context: action evaluation: `queue` ignored, already in expected state: run/success -> success [action: queue] [check_name: Rule: queueit (queue)] [conclusion: success] [gh_author: mergify-sileht[bot]] [gh_branch: 20231018114631/test_freeze_eventlogs/main] [gh_owner: mergifyio-testing] [gh_private: False] [gh_pull: 33773] [gh_pull_base_sha: d9b44ca14ce953bf19608307ae8e0f9eaad0083e] [gh_pull_head_owner: mergifyio-testing] [gh_pull_head_ref: 20231018114631/test_freeze_eventlogs/integration/pr1] [gh_pull_head_sha: 8367f370bd775373f19a7251a28a0ec5ed9de202] [gh_pull_locked: False] [gh_pull_merge_commit_sha: 4b0471b042006241d0f38f9d8d44e1657d39ed32] [gh_pull_mergeable: None] [gh_pull_mergeable_state: merged] [gh_pull_url: https://github.com/mergifyio-testing/functional-testing-repo-sileht/pull/33773] [gh_repo: functional-testing-repo-sileht] [previous_conclusion: success] [report: Result(conclusion=<Conclusion.SUCCESS: 'success'>, title='Already in expected state', summary='', annotations=None, started_at=None, ended_at=None, log_details=None, details_url=None)] [rule_summary: - [X] `-draft` [:pushpin: queue requirement]
- [X] `-mergify-configuration-changed` [:pushpin: queue -> allow_merging_configuration_change setting requirement]
- [X] `base=20231018114631/test_freeze_eventlogs/main`
- [X] `label=queue`
- [X] any of: [:twisted_rightwards_arrows: queue conditions]
  - [X] all of [:pushpin: queue conditions of queue `default`]] [worker_id: shared-0]
  INFO     Context: summary unchanged [conclusions: {'Rule: queueit (queue)': <Conclusion.SUCCESS: 'success'>}] [gh_author: mergify-sileht[bot]] [gh_branch: 20231018114631/test_freeze_eventlogs/main] [gh_owner: mergifyio-testing] [gh_private: False] [gh_pull: 33773] [gh_pull_base_sha: d9b44ca14ce953bf19608307ae8e0f9eaad0083e] [gh_pull_head_owner: mergifyio-testing] [gh_pull_head_ref: 20231018114631/test_freeze_eventlogs/integration/pr1] [gh_pull_head_sha: 8367f370bd775373f19a7251a28a0ec5ed9de202] [gh_pull_locked: False] [gh_pull_merge_commit_sha: 4b0471b042006241d0f38f9d8d44e1657d39ed32] [gh_pull_mergeable: None] [gh_pull_mergeable_state: merged] [gh_pull_url: https://github.com/mergifyio-testing/functional-testing-repo-sileht/pull/33773] [gh_repo: functional-testing-repo-sileht] [previous_conclusions: {'Rule: queueit (queue)': <Conclusion.SUCCESS: 'success'>}] [worker_id: shared-0]
  INFO     mergify_engine.worker.task: tasks stopping [count: 3] [tasks: ['dedicated-workers-cache-syncer', 'dedicated-workers-spawner', 'gitter']] [worker_id: dedicated-40527191]
  INFO     mergify_engine.worker.task: dedicated-workers-spawner task exited [worker_id: dedicated-40527191]
  INFO     mergify_engine.worker.task: gitter task exited [worker_id: dedicated-40527191]
  INFO     mergify_engine.worker.task: tasks stopped [count: 3] [tasks: ['dedicated-workers-cache-syncer', 'dedicated-workers-spawner', 'gitter']] [worker_id: dedicated-40527191]
  TEST     mergify_engine.tests.functional.base: WAITING FOR pull_request/closed: {'action': 'closed'} [worker_id: dedicated-40527191]
  INFO     mergify_engine.worker.task: dedicated-workers-cache-syncer task exited [worker_id: dedicated-40527191]
  TEST     mergify_engine.tests.functional.base: EVENT RECEIVED check_run/created/in_progress/None: {'hook_id': '365596432', 'type': 'check_run', 'payload': {'action': 'created', 'check_run': {'name': 'Rule: queueit (queue)', 'head_sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'status': 'in_progress', 'conclusion': None, 'output': {'title': 'The pull request is the 1st in the queue to be merged', 'summary': '#33773 is queued for merge.\n\nRequired conditions of queue `default` for merge:\n\n- [ ] `status-success=continuous-integration/fake-ci`\n\nRequired conditions to stay in the queue:\n\n- [X] `-draft` [:pushpin: queue requirement]\n- [X] `-mergify-configuration-changed` [:pushpin: queue -> allow_merging_configuration_change setting requirement]\n- [X] `base=20231018114631/test_freeze_eventlogs/main`\n- [X] `label=queue`\n- [X] any of: [:twisted_rightwards_arrows: queue conditions]\n  - [X] all of [:pushpin: queue conditions of queue `default`]\n\nVisit the [Mergify Dashboard](http://localhost:3000/github/mergifyio-testing/repo/functional-testing-repo-sileht/queues/partitions/__default__?queues=default&branch=20231018114631%2Ftest_freeze_eventlogs%2Fmain) to check the state of the queue `default`.', 'text': None, 'annotations_count': 0}, 'check_suite': {'head_branch': '20231018114631/test_freeze_eventlogs/integration/pr1', 'head_sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'status': 'in_progress', 'conclusion': None, 'pull_requests': [{'number': 33773, 'head': {'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht'}}, 'base': {'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht'}}}]}, 'pull_requests': [{'number': 33773, 'head': {'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht'}}, 'base': {'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht'}}}]}, 'organization': {'login': 'mergifyio-testing'}}} [worker_id: dedicated-40527191]
  INFO     mergify_engine.github_events: GitHubApp event ignored [event_id: 123456789] [event_type: check_run] [gh_owner: mergifyio-testing] [gh_repo: functional-testing-repo-sileht] [reason: mergify check_run] [sender: mergify-test1] [worker_id: dedicated-40527191]
  TEST     mergify_engine.tests.functional.base: EVENT RECEIVED check_run/completed/completed/success: {'hook_id': '365596432', 'type': 'check_run', 'payload': {'action': 'completed', 'check_run': {'name': 'Summary', 'head_sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'status': 'completed', 'conclusion': 'success', 'output': {'title': '1 rule matches', 'summary': '<!-- eydSdWxlOiBxdWV1ZWl0IChxdWV1ZSknOiBudWxsfQo= -->\n### Rule: queueit (queue)\n- [X] `-draft` [:pushpin: queue requirement]\n- [X] `-mergify-configuration-changed` [:pushpin: queue -> allow_merging_configuration_change setting requirement]\n- [X] `base=20231018114631/test_freeze_eventlogs/main`\n- [X] `label=queue`\n- [X] any of: [:twisted_rightwards_arrows: queue conditions]\n  - [X] all of [:pushpin: queue conditions of queue `default`]\n\n<hr />\n\n<details>\n<summary>Mergify commands and options</summary>\n\n<br />\n\nMore conditions and actions can be found in the [documentation](https://docs.mergify.com/).\n\nYou can also trigger Mergify actions by commenting on this pull request:\n\n- `@Mergifyio refresh` will re-evaluate the rules\n- `@Mergifyio rebase` will rebase this PR on its base branch\n- `@Mergifyio update` will merge the base branch into this PR\n- `@Mergifyio backport <destination>` will backport this PR on `<destination>` branch\n\nAdditionally, on Mergify [dashboard](http://localhost:3000) you can:\n\n- look at your merge queues\n- generate the Mergify configuration with the config editor.\n\nFinally, you can contact us on https://mergify.com\n</details>\n', 'text': None, 'annotations_count': 0}, 'check_suite': {'head_branch': '20231018114631/test_freeze_eventlogs/integration/pr1', 'head_sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'status': 'in_progress', 'conclusion': None, 'pull_requests': [{'number': 33773, 'head': {'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht'}}, 'base': {'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht'}}}]}, 'pull_requests': [{'number': 33773, 'head': {'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht'}}, 'base': {'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht'}}}]}, 'organization': {'login': 'mergifyio-testing'}}} [worker_id: dedicated-40527191]
  INFO     mergify_engine.github_events: GitHubApp event ignored [event_id: 123456789] [event_type: check_run] [gh_owner: mergifyio-testing] [gh_repo: functional-testing-repo-sileht] [reason: mergify check_run] [sender: mergify-test1] [worker_id: dedicated-40527191]
  TEST     mergify_engine.tests.functional.base: EVENT RECEIVED check_run/created/completed/success: {'hook_id': '365596432', 'type': 'check_run', 'payload': {'action': 'created', 'check_run': {'name': 'Summary', 'head_sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'status': 'completed', 'conclusion': 'success', 'output': {'title': '1 rule matches', 'summary': '<!-- eydSdWxlOiBxdWV1ZWl0IChxdWV1ZSknOiBudWxsfQo= -->\n### Rule: queueit (queue)\n- [X] `-draft` [:pushpin: queue requirement]\n- [X] `-mergify-configuration-changed` [:pushpin: queue -> allow_merging_configuration_change setting requirement]\n- [X] `base=20231018114631/test_freeze_eventlogs/main`\n- [X] `label=queue`\n- [X] any of: [:twisted_rightwards_arrows: queue conditions]\n  - [X] all of [:pushpin: queue conditions of queue `default`]\n\n<hr />\n\n<details>\n<summary>Mergify commands and options</summary>\n\n<br />\n\nMore conditions and actions can be found in the [documentation](https://docs.mergify.com/).\n\nYou can also trigger Mergify actions by commenting on this pull request:\n\n- `@Mergifyio refresh` will re-evaluate the rules\n- `@Mergifyio rebase` will rebase this PR on its base branch\n- `@Mergifyio update` will merge the base branch into this PR\n- `@Mergifyio backport <destination>` will backport this PR on `<destination>` branch\n\nAdditionally, on Mergify [dashboard](http://localhost:3000) you can:\n\n- look at your merge queues\n- generate the Mergify configuration with the config editor.\n\nFinally, you can contact us on https://mergify.com\n</details>\n', 'text': None, 'annotations_count': 0}, 'check_suite': {'head_branch': '20231018114631/test_freeze_eventlogs/integration/pr1', 'head_sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'status': 'in_progress', 'conclusion': None, 'pull_requests': [{'number': 33773, 'head': {'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht'}}, 'base': {'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht'}}}]}, 'pull_requests': [{'number': 33773, 'head': {'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht'}}, 'base': {'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht'}}}]}, 'organization': {'login': 'mergifyio-testing'}}} [worker_id: dedicated-40527191]
  INFO     mergify_engine.github_events: GitHubApp event ignored [event_id: 123456789] [event_type: check_run] [gh_owner: mergifyio-testing] [gh_repo: functional-testing-repo-sileht] [reason: mergify check_run] [sender: mergify-test1] [worker_id: dedicated-40527191]
  TEST     mergify_engine.tests.functional.base: EVENT RECEIVED check_run/created/in_progress/None: {'hook_id': '365596432', 'type': 'check_run', 'payload': {'action': 'created', 'check_run': {'name': 'Queue: Embarked in merge queue', 'head_sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'status': 'in_progress', 'conclusion': None, 'output': {'title': 'The pull request is embarked with 20231018114631/test_freeze_eventlogs/main (d9b44ca) for merge', 'summary': '<!-- eyJvdXRjb21lIjogeyJfX3B5dHlwZV9fIjogImVudW0iLCAiY2xhc3MiOiAiVHJhaW5DYXJPdXRjb21lIiwgIm5hbWUiOiAiVU5LTk9XTiJ9LCAib3V0Y29tZV9tZXNzYWdlIjogbnVsbH0= -->\n\nRequired conditions for merge:\n\n- [ ] `status-success=continuous-integration/fake-ci`', 'text': None, 'annotations_count': 0}, 'check_suite': {'head_branch': '20231018114631/test_freeze_eventlogs/integration/pr1', 'head_sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'status': 'in_progress', 'conclusion': None, 'pull_requests': [{'number': 33773, 'head': {'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht'}}, 'base': {'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht'}}}]}, 'pull_requests': [{'number': 33773, 'head': {'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht'}}, 'base': {'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht'}}}]}, 'organization': {'login': 'mergifyio-testing'}}} [worker_id: dedicated-40527191]
  INFO     mergify_engine.github_events: GitHubApp event ignored [event_id: 123456789] [event_type: check_run] [gh_owner: mergifyio-testing] [gh_repo: functional-testing-repo-sileht] [reason: mergify check_run] [sender: mergify-test1] [worker_id: dedicated-40527191]
  TEST     mergify_engine.tests.functional.base: EVENT RECEIVED check_run/completed/completed/neutral: {'hook_id': '365596432', 'type': 'check_run', 'payload': {'action': 'completed', 'check_run': {'name': 'Queue: Embarked in merge queue', 'head_sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'status': 'completed', 'conclusion': 'neutral', 'output': {'title': 'The pull request 33773 is in queue', 'summary': 'The pull request 33773 is in queue, this summary will be updated once the queue checks have started.', 'text': None, 'annotations_count': 0}, 'check_suite': {'head_branch': '20231018114631/test_freeze_eventlogs/integration/pr1', 'head_sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'status': 'in_progress', 'conclusion': None, 'pull_requests': [{'number': 33773, 'head': {'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht'}}, 'base': {'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht'}}}]}, 'pull_requests': [{'number': 33773, 'head': {'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht'}}, 'base': {'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht'}}}]}, 'organization': {'login': 'mergifyio-testing'}}} [worker_id: dedicated-40527191]
  INFO     mergify_engine.github_events: GitHubApp event ignored [event_id: 123456789] [event_type: check_run] [gh_owner: mergifyio-testing] [gh_repo: functional-testing-repo-sileht] [reason: mergify check_run] [sender: mergify-test1] [worker_id: dedicated-40527191]
  TEST     mergify_engine.tests.functional.base: EVENT RECEIVED check_run/completed/completed/success: {'hook_id': '365596432', 'type': 'check_run', 'payload': {'action': 'completed', 'check_run': {'name': 'Queue: Embarked in merge queue', 'head_sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'status': 'completed', 'conclusion': 'success', 'output': {'title': 'The pull request embarked with 20231018114631/test_freeze_eventlogs/main (d9b44ca) is mergeable', 'summary': '<!-- eyJvdXRjb21lIjogeyJfX3B5dHlwZV9fIjogImVudW0iLCAiY2xhc3MiOiAiVHJhaW5DYXJPdXRjb21lIiwgIm5hbWUiOiAiTUVSR0VBQkxFIn0sICJvdXRjb21lX21lc3NhZ2UiOiBudWxsfQ== -->\n\n❄️ The merge is currently blocked by the freeze of the queue `default`, for the following reason: `test freeze reason` ❄️\n\nRequired conditions for merge:\n\n- [X] `status-success=continuous-integration/fake-ci`\n- [X] all of [📃 From pull request rule queueit]:\n  - `-draft` [:pushpin: queue requirement]\n    - [X] #33773\n  - `-mergify-configuration-changed` [:pushpin: queue -> allow_merging_configuration_change setting requirement]\n    - [X] #33773\n  - [X] `base=20231018114631/test_freeze_eventlogs/main`\n  - `label=queue`\n    - [X] #33773\n  - [X] any of [:twisted_rightwards_arrows: queue conditions]:\n    - [X] all of [:pushpin: queue conditions of queue `default`]\n\nCheck-runs and statuses of the embarked pull request #33773:\n\n<table><tr><td align="center" width="48" height="48"><img src="https://raw.githubusercontent.com/Mergifyio/mergify/master/assets/check-green-16.png" width="16" height="16" /></td><td align="center" width="48" height="48"><img src="https://avatars.githubusercontent.com/u/40527191?v=4&s=40" width="16" height="16" /></td><td><b>continuous-integration/fake-ci</b>The CI is success</td><td><a href="">details</a></td></tr></table>\n', 'text': None, 'annotations_count': 0}, 'check_suite': {'head_branch': '20231018114631/test_freeze_eventlogs/integration/pr1', 'head_sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'status': 'in_progress', 'conclusion': None, 'pull_requests': [{'number': 33773, 'head': {'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht'}}, 'base': {'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht'}}}]}, 'pull_requests': [{'number': 33773, 'head': {'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht'}}, 'base': {'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht'}}}]}, 'organization': {'login': 'mergifyio-testing'}}} [worker_id: dedicated-40527191]
  INFO     mergify_engine.github_events: GitHubApp event ignored [event_id: 123456789] [event_type: check_run] [gh_owner: mergifyio-testing] [gh_repo: functional-testing-repo-sileht] [reason: mergify check_run] [sender: mergify-test1] [worker_id: dedicated-40527191]
  TEST     mergify_engine.tests.functional.base: EVENT RECEIVED check_run/completed/completed/success: {'hook_id': '365596432', 'type': 'check_run', 'payload': {'action': 'completed', 'check_run': {'name': 'Queue: Embarked in merge queue', 'head_sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'status': 'completed', 'conclusion': 'success', 'output': {'title': 'The pull request embarked with 20231018114631/test_freeze_eventlogs/main (d9b44ca) is mergeable', 'summary': '<!-- eyJvdXRjb21lIjogeyJfX3B5dHlwZV9fIjogImVudW0iLCAiY2xhc3MiOiAiVHJhaW5DYXJPdXRjb21lIiwgIm5hbWUiOiAiTUVSR0VBQkxFIn0sICJvdXRjb21lX21lc3NhZ2UiOiBudWxsfQ== -->\n\nRequired conditions for merge:\n\n- [X] `status-success=continuous-integration/fake-ci`\n- [X] all of [📃 From pull request rule queueit]:\n  - `-draft` [:pushpin: queue requirement]\n    - [X] #33773\n  - `-mergify-configuration-changed` [:pushpin: queue -> allow_merging_configuration_change setting requirement]\n    - [X] #33773\n  - [X] `base=20231018114631/test_freeze_eventlogs/main`\n  - `label=queue`\n    - [X] #33773\n  - [X] any of [:twisted_rightwards_arrows: queue conditions]:\n    - [X] all of [:pushpin: queue conditions of queue `default`]\n\nCheck-runs and statuses of the embarked pull request #33773:\n\n<table><tr><td align="center" width="48" height="48"><img src="https://raw.githubusercontent.com/Mergifyio/mergify/master/assets/check-green-16.png" width="16" height="16" /></td><td align="center" width="48" height="48"><img src="https://avatars.githubusercontent.com/u/40527191?v=4&s=40" width="16" height="16" /></td><td><b>continuous-integration/fake-ci</b>The CI is success</td><td><a href="">details</a></td></tr></table>\n', 'text': None, 'annotations_count': 0}, 'check_suite': {'head_branch': '20231018114631/test_freeze_eventlogs/integration/pr1', 'head_sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'status': 'in_progress', 'conclusion': None, 'pull_requests': [{'number': 33773, 'head': {'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht'}}, 'base': {'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht'}}}]}, 'pull_requests': [{'number': 33773, 'head': {'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht'}}, 'base': {'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht'}}}]}, 'organization': {'login': 'mergifyio-testing'}}} [worker_id: dedicated-40527191]
  INFO     mergify_engine.github_events: GitHubApp event ignored [event_id: 123456789] [event_type: check_run] [gh_owner: mergifyio-testing] [gh_repo: functional-testing-repo-sileht] [reason: mergify check_run] [sender: mergify-test1] [worker_id: dedicated-40527191]
  TEST     mergify_engine.tests.functional.base: EVENT RECEIVED pull_request/closed: {'hook_id': '365596432', 'type': 'pull_request', 'payload': {'action': 'closed', 'number': 33773, 'pull_request': {'number': 33773, 'state': 'closed', 'locked': False, 'title': 'test_freeze_eventlogs: pull request n1 from integration', 'merge_commit_sha': '4b0471b042006241d0f38f9d8d44e1657d39ed32', 'assignee': None, 'assignees':  , 'requested_reviewers':  , 'requested_teams':  , 'labels': [{'name': 'queue', 'color': '000000', 'default': False, 'description': None}], 'milestone': None, 'draft': False, 'head': {'label': 'mergifyio-testing:20231018114631/test_freeze_eventlogs/integration/pr1', 'ref': '20231018114631/test_freeze_eventlogs/integration/pr1', 'sha': '8367f370bd775373f19a7251a28a0ec5ed9de202', 'repo': {'name': 'functional-testing-repo-sileht', 'full_name': 'mergifyio-testing/functional-testing-repo-sileht', 'private': False, 'owner': {'login': 'mergifyio-testing', 'gravatar_id': '', 'type': 'Organization', 'site_admin': False}, 'description': None, 'fork': False, 'homepage': None, 'size': 210, 'stargazers_count': 0, 'watchers_count': 0, 'language': None, 'has_issues': True, 'has_projects': True, 'has_downloads': True, 'has_wiki': True, 'has_pages': False, 'has_discussions': False, 'forks_count': 2, 'archived': False, 'disabled': False, 'open_issues_count': 0, 'license': None, 'allow_forking': True, 'is_template': False, 'web_commit_signoff_required': False, 'topics':  , 'visibility': 'public', 'forks': 2, 'open_issues': 0, 'watchers': 0, 'default_branch': 'main', 'allow_squash_merge': True, 'allow_merge_commit': True, 'allow_rebase_merge': True, 'allow_auto_merge': False, 'delete_branch_on_merge': False, 'allow_update_branch': True, 'use_squash_pr_title_as_default': False, 'squash_merge_commit_message': 'COMMIT_MESSAGES', 'squash_merge_commit_title': 'COMMIT_OR_PR_TITLE', 'merge_commit_message': 'PR_TITLE', 'merge_commit_title': 'MERGE_MESSAGE'}}, 'base': {'label': 'mergifyio-testing:20231018114631/test_freeze_eventlogs/main', 'ref': '20231018114631/test_freeze_eventlogs/main', 'sha': 'd9b44ca14ce953bf19608307ae8e0f9eaad0083e', 'repo': {'name': 'functional-testing-repo-sileht', 'full_name': 'mergifyio-testing/functional-testing-repo-sileht', 'private': False, 'owner': {'login': 'mergifyio-testing', 'gravatar_id': '', 'type': 'Organization', 'site_admin': False}, 'description': None, 'fork': False, 'homepage': None, 'size': 210, 'stargazers_count': 0, 'watchers_count': 0, 'language': None, 'has_issues': True, 'has_projects': True, 'has_downloads': True, 'has_wiki': True, 'has_pages': False, 'has_discussions': False, 'forks_count': 2, 'archived': False, 'disabled': False, 'open_issues_count': 0, 'license': None, 'allow_forking': True, 'is_template': False, 'web_commit_signoff_required': False, 'topics':  , 'visibility': 'public', 'forks': 2, 'open_issues': 0, 'watchers': 0, 'default_branch': 'main', 'allow_squash_merge': True, 'allow_merge_commit': True, 'allow_rebase_merge': True, 'allow_auto_merge': False, 'delete_branch_on_merge': False, 'allow_update_branch': True, 'use_squash_pr_title_as_default': False, 'squash_merge_commit_message': 'COMMIT_MESSAGES', 'squash_merge_commit_title': 'COMMIT_OR_PR_TITLE', 'merge_commit_message': 'PR_TITLE', 'merge_commit_title': 'MERGE_MESSAGE'}}, 'author_association': 'CONTRIBUTOR', 'auto_merge': None, 'active_lock_reason': None, 'merged': True, 'mergeable': None, 'rebaseable': None, 'mergeable_state': 'unknown', 'merged_by': {'login': 'mergify-sileht[bot]', 'gravatar_id': '', 'type': 'Bot', 'site_admin': False}, 'comments': 0, 'review_comments': 0, 'maintainer_can_modify': False, 'commits': 1, 'additions': 0, 'deletions': 0, 'changed_files': 1}, 'organization': {'login': 'mergifyio-testing'}}} [worker_id: dedicated-40527191]
  INFO     mergify_engine.github_events: GitHubApp event pushed [event_id: 123456789] [event_type: pull_request] [gh_owner: mergifyio-testing] [gh_pull: 33773] [gh_repo: functional-testing-repo-sileht] [priority: None] [sender: mergify-sileht[bot]] [slim_event: {'action': 'closed', 'sender': {'login': 'mergify-sileht[bot]', 'id': 108276933, 'type': 'Bot'}, 'delivery_id': '123456789', 'received_at': '2023-10-20T09:16:28.363754+00:00'}] [worker_id: dedicated-40527191]
  INFO     mergify_engine.tests.functional.base: gitter messages [messages: [('git directory created', {'path': '/tmp/mergify-gitterzlt40p27', 'version': 'git version 2.39.3 (Apple Git-145)\n'}), ('cleaning: /tmp/mergify-gitterzlt40p27', {})]] [worker_id: dedicated-40527191]
 slowest 10 durations
10.17s call     mergify_engine/tests/functional/api/test_partitions.py::TestPartitionsApi::test_estimated_time_of_merge_normal_partitions
5.38s call     mergify_engine/tests/functional/api/test_queue_freeze.py::TestQueueFreeze::test_create_queue_freeze_without_cascading_effect
4.75s call     mergify_engine/tests/functional/actions/test_queue_train_api_calls.py::TestTrainApiCalls::test_train_car_state_waiting_for_schedule_after_pr_ahead_dequeued
4.66s call     mergify_engine/tests/functional/commands/test_unqueue.py::TestUnQueueCommand::test_unqueue_pr_in_2_partitions
4.24s call     mergify_engine/tests/functional/actions/test_queue.py::TestQueueAction::test_unfreeze_outside_schedule
3.49s call     mergify_engine/tests/functional/actions/test_queue_with_partition_rules.py::TestQueueWithPartitionRules::test_partition_reset_when_pr_manual_merge_matches_one_partition
3.20s setup    mergify_engine/tests/unit/api/test_events.py::test_api_links_with_query_params
3.06s call     mergify_engine/tests/functional/commands/test_queue.py::TestQueueCommand::test_queue_command_unqueue_pr_after_ci_failure
2.87s call     mergify_engine/tests/functional/actions/test_queue.py::TestQueueAction::test_draft_pr_train_reset_after_unexpected_base_branch_changes
2.81s call     mergify_engine/tests/functional/api/test_eventlogs.py::TestEventLogsAction::test_freeze_eventlogs
 short test summary info
FAILED mergify_engine/tests/functional/actions/test_dismiss_review.py::TestDismissReviewsAction::test_dismiss_reviews - AssertionError: assert {'events':  ...0, 'total': 0} == {'events': [{...2, 'total': 2}
  Omitting 1 identical items, use -vv to show
  Differing items:
  {'size': 0} != {'size': 2}
  {'total': 0} != {'total': 2}
  {'events':  } != {'events': [{'event': 'action.dismiss_reviews', 'id': ANY_INT, 'metadata': {'users': ['mergify-test1']}, 'pull_request...vent': 'action.dismiss_reviews', 'id': ANY_INT, 'metadata': {'users': ['mergify-test1']}, 'pull_request': 33929, ...}]}
  Full diff:
    {
  +  'events':  ,
  -  'events': [{'event': 'action.dismiss_reviews',
  -              'id': ANY_INT,
  -              'metadata': {'users': ['mergify-test1']},
  -              'pull_request': 33929,
  -              'received_at': ANY_AWARE_DATETIME_STR,
  -              'repository': 'mergifyio-testing/functional-testing-repo-sileht',
  -              'timestamp': ANY_AWARE_DATETIME_STR,
  -              'trigger': 'Rule: dismiss reviews',
  -              'type': 'action.dismiss_reviews'},
  -             {'event': 'action.dismiss_reviews',
  -              'id': ANY_INT,
  -              'metadata': {'users': ['mergify-test1']},
  -              'pull_request': 33929,
  -              'received_at': ANY_AWARE_DATETIME_STR,
  -              'repository': 'mergifyio-testing/functional-testing-repo-sileht',
  -              'timestamp': ANY_AWARE_DATETIME_STR,
  -              'trigger': 'Rule: dismiss reviews',
  -              'type': 'action.dismiss_reviews'}],
     'per_page': 10,
  -  'size': 2,
  ?          ^
  +  'size': 0,
  ?          ^
  -  'total': 2,
  ?           ^
  +  'total': 0,
  ?           ^
    }
FAILED mergify_engine/tests/functional/actions/test_queue.py::TestQueueAction::test_draft_pr_train_reset_after_unexpected_base_branch_changes - Failed: No action.queue.checks_end event found
FAILED mergify_engine/tests/functional/actions/test_queue.py::TestQueueAction::test_pr_requeue_after_pr_unexpected_updated_changes - AssertionError: Event action.queue.checks_end not found
FAILED mergify_engine/tests/functional/actions/test_queue.py::TestQueueAction::test_queue_checks_end_emission - assert 0 == 1
 +  where 0 = <bound method Result.scalar of <sqlalchemy.engine.result.ChunkedIteratorResult object at 0x7f239773f7d0>>()
 +    where <bound method Result.scalar of <sqlalchemy.engine.result.ChunkedIteratorResult object at 0x7f239773f7d0>> = <sqlalchemy.engine.result.ChunkedIteratorResult object at 0x7f239773f7d0>.scalar
FAILED mergify_engine/tests/functional/api/test_eventlogs.py::TestEventLogsAction::test_freeze_eventlogs - AssertionError: assert {'events':  ...0, 'total': 0} == {'events': [{...5, 'total': 5}
  Omitting 1 identical items, use -vv to show
  Differing items:
  {'size': 0} != {'size': 5}
  {'total': 0} != {'total': 5}
  {'events':  } != {'events': [{'event': 'action.queue.merged', 'id': ANY_INT, 'metadata': {'branch': '20231018114631/test_freeze_eventlo...ogs/main', 'partition_name': '__default__', 'position': 0, 'queue_name': 'default', ...}, 'pull_request': 33773, ...}]}
  Full diff:
    {
  +  'events':  ,
  -  'events': [{'event': 'action.queue.merged',
  -              'id': ANY_INT,
  -              'metadata': {'branch': '20231018114631/test_freeze_eventlogs/main',
  -                           'partition_names': ['__default__'],
  -                           'queue_name': 'default',
  -                           'queued_at': <ANY>},
  -              'pull_request': 33773,
  -              'received_at': ANY_AWARE_DATETIME_STR,
  -              'repository': 'mergifyio-testing/functional-testing-repo-sileht',
  -              'timestamp': ANY_AWARE_DATETIME_STR,
  -              'trigger': 'Rule: queueit',
  -              'type': 'action.queue.merged'},
  -             {'event': 'action.queue.leave',
  -              'id': ANY_INT,
  -              'metadata': {'branch': '20231018114631/test_freeze_eventlogs/main',
  -                           'merged': True,
  -                           'partition_name': '__default__',
  -                           'position': 0,
  -                           'queue_name': 'default',
  -                           'queued_at': <ANY>,
  -                           'reason': 'Pull request #33773 has been merged '
  -                                     'automatically at '
  -                                     '*4b0471b042006241d0f38f9d8d44e1657d39ed32*',
  -                           'seconds_waiting_for_freeze': <ANY>,
  -                           'seconds_waiting_for_schedule': 0},
  -              'pull_request': 33773,
  -              'received_at': ANY_AWARE_DATETIME_STR,
  -              'repository': 'mergifyio-testing/functional-testing-repo-sileht',
  -              'timestamp': ANY_AWARE_DATETIME_STR,
  -              'trigger': 'Rule: queueit',
  -              'type': 'action.queue.leave'},
  -             {'event': 'action.queue.checks_end',
  -              'id': ANY_INT,
  -              'metadata': {'abort_code': None,
  -                           'abort_reason': '',
  -                           'abort_status': 'DEFINITIVE',
  -                           'aborted': False,
  -                           'branch': '20231018114631/test_freeze_eventlogs/main',
  -                           'partition_name': '__default__',
  -                           'position': 0,
  -                           'queue_name': 'default',
  -                           'queued_at': <ANY>,
  -                           'speculative_check_pull_request': {'checks_conclusion': 'success',
  -                                                              'checks_ended_at': <ANY>,
  -                                                              'checks_started_at': <ANY>,
  -                                                              'checks_timed_out': False,
  -                                                              'in_place': True,
  -                                                              'number': 33773,
  -                                                              'unsuccessful_checks':  },
  -                           'unqueue_code': None},
  -              'pull_request': 33773,
  -              'received_at': ANY_AWARE_DATETIME_STR,
  -              'repository': 'mergifyio-testing/functional-testing-repo-sileht',
  -              'timestamp': ANY_AWARE_DATETIME_STR,
  -              'trigger': 'merge queue internal',
  -              'type': 'action.queue.checks_end'},
  -             {'event': 'action.queue.checks_start',
  -              'id': ANY_INT,
  -              'metadata': {'branch': '20231018114631/test_freeze_eventlogs/main',
  -                           'partition_name': '__default__',
  -                           'position': 0,
  -                           'queue_name': 'default',
  -                           'queued_at': <ANY>,
  -                           'speculative_check_pull_request': {'checks_conclusion': 'pending',
  -                                                              'checks_ended_at': None,
  -                                                              'checks_started_at': None,
  -                                                              'checks_timed_out': False,
  -                                                              'in_place': True,
  -                                                              'number': 33773,
  -                                                              'unsuccessful_checks':  },
  -                           'start_reason': 'First time checking pull request '
  -                                           '#33773'},
  -              'pull_request': 33773,
  -              'received_at': ANY_AWARE_DATETIME_STR,
  -              'repository': 'mergifyio-testing/functional-testing-repo-sileht',
  -              'timestamp': ANY_AWARE_DATETIME_STR,
  -              'trigger': 'merge queue internal',
  -              'type': 'action.queue.checks_start'},
  -             {'event': 'action.queue.enter',
  -              'id': ANY_INT,
  -              'metadata': {'branch': '20231018114631/test_freeze_eventlogs/main',
  -                           'partition_name': '__default__',
  -                           'position': 0,
  -                           'queue_name': 'default',
  -                           'queued_at': <ANY>},
  -              'pull_request': 33773,
  -              'received_at': ANY_AWARE_DATETIME_STR,
  -              'repository': 'mergifyio-testing/functional-testing-repo-sileht',
  -              'timestamp': ANY_AWARE_DATETIME_STR,
  -              'trigger': 'Rule: queueit',
  -              'type': 'action.queue.enter'}],
     'per_page': 10,
  -  'size': 5,
  ?          ^
  +  'size': 0,
  ?          ^
  -  'total': 5,
  ?           ^
  +  'total': 0,
  ?           ^
    }
ERROR mergify_engine/tests/functional/test_attributes.py::TestAttributesWithSub::test_check_run_timeout - AssertionError: assert   == ['failed to r...l: eventlogs']
  Right contains 2 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
  - ['failed to run signal: eventlogs', 'failed to run signal: eventlogs']
  +  
ERROR mergify_engine/tests/functional/test_attributes.py::TestAttributesWithSub::test_linear_history - AssertionError: assert   == ['failed to r...l: eventlogs']
  Right contains one more item: 'failed to run signal: eventlogs'
  Full diff:
  - ['failed to run signal: eventlogs']
  +  
ERROR mergify_engine/tests/functional/actions/test_dismiss_review.py::TestDismissReviewsAction::test_dismiss_reviews - AssertionError: assert   == ['failed to r...l: eventlogs']
  Right contains 2 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
  - ['failed to run signal: eventlogs', 'failed to run signal: eventlogs']
  +  
ERROR mergify_engine/tests/functional/actions/test_queue.py::TestQueueAction::test_draft_pr_train_reset_after_unexpected_base_branch_changes - AssertionError: assert   == ['failed to r...entlogs', ...]
  Right contains 8 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/actions/test_queue.py::TestQueueAction::test_pr_requeue_after_pr_unexpected_updated_changes - AssertionError: assert   == ['failed to r...l: eventlogs']
  Right contains 6 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/actions/test_queue.py::TestQueueAction::test_queue_checks_end_emission - AssertionError: assert   == ['failed to r...l: eventlogs']
  Right contains 4 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/actions/test_queue.py::TestQueueAction::test_queue_inplace_autosquash_false - AssertionError: assert   == ['failed to r...l: eventlogs']
  Right contains 2 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
  - ['failed to run signal: eventlogs', 'failed to run signal: eventlogs']
  +  
ERROR mergify_engine/tests/functional/actions/test_queue.py::TestQueueAction::test_queue_just_rebase - AssertionError: assert   == ['failed to r...l: eventlogs']
  Right contains 5 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/actions/test_queue.py::TestQueueAction::test_queue_rule_deleted - AssertionError: assert   == ['failed to r...l: eventlogs']
  Right contains 4 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/actions/test_queue.py::TestQueueAction::test_queue_with_rebase_on_githubapp_pr - AssertionError: assert   == ['failed to r...l: eventlogs']
  Right contains 2 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
  - ['failed to run signal: eventlogs', 'failed to run signal: eventlogs']
  +  
ERROR mergify_engine/tests/functional/actions/test_queue.py::TestQueueAction::test_target_branch_vanished - AssertionError: assert   == ['failed to r...l: eventlogs']
  Right contains 6 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/actions/test_queue.py::TestQueueAction::test_unfreeze_outside_schedule - AssertionError: assert   == ['failed to r...entlogs', ...]
  Right contains 7 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/actions/test_queue_train_api_calls.py::TestTrainApiCalls::test_train_car_state_waiting_for_schedule_after_pr_ahead_dequeued - AssertionError: assert   == ['failed to r...entlogs', ...]
  Right contains 12 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/actions/test_queue_with_partition_rules.py::TestQueueWithPartitionRules::test_partition_reset_when_pr_manual_merge_matches_one_partition - AssertionError: assert   == ['failed to r...l: eventlogs']
  Right contains 6 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/actions/test_queue_with_partition_rules.py::TestQueueWithPartitionRules::test_queue_1_pr_multiple_partitions_match_no_inplace - AssertionError: assert   == ['failed to r...entlogs', ...]
  Right contains 9 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/api/test_eventlogs.py::TestEventLogsAction::test_freeze_eventlogs - AssertionError: assert   == ['failed to r...entlogs', ...]
  Right contains 8 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/api/test_partitions.py::TestPartitionsApi::test_estimated_time_of_merge_normal_partitions - AssertionError: assert   == ['failed to r...entlogs', ...]
  Right contains 24 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/api/test_queue_freeze.py::TestQueueFreeze::test_create_queue_freeze_without_cascading_effect - AssertionError: assert   == ['failed to r...entlogs', ...]
  Right contains 14 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/commands/test_queue.py::TestQueueCommand::test_command_queue_non_matching_queue_conditions_and_specifying_queue_name - AssertionError: assert   == ['failed to r...l: eventlogs']
  Right contains 5 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/commands/test_queue.py::TestQueueCommand::test_queue_command_unqueue_pr_after_ci_failure - AssertionError: assert   == ['failed to r...l: eventlogs']
  Right contains 4 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
ERROR mergify_engine/tests/functional/commands/test_unqueue.py::TestUnQueueCommand::test_unqueue_pr_in_2_partitions - AssertionError: assert   == ['failed to r...entlogs', ...]
  Right contains 9 more items, first extra item: 'failed to run signal: eventlogs'
  Full diff:
    [
  +  ,
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
  -  'failed to run signal: eventlogs',
    ]
 5 failed, 196 passed, 1804 deselected, 21 errors in 239.36s (0:03:59)
##[error]Process completed with exit code 1.